### ISSUE TITLE
Song of Heiligenstadt [PCSG01070]

### ISSUE THREAD

**Issue by othmanead** [Vita3K/compatibility#674]
# App summary
- App name: Song of Heiligenstadt
- App serial: PCSG01070
- App version: 1.0

# Vita3K summary
- Version: v0.1.5
- Build number: 2647
- Commit hash: https://github.com/vita3k/vita3k/commit/d2e99945
- CPU backend: Dynamic  

# Test environment summary
- Tested by: othmanead
- OS: Windows 11
- CPU: Intel i5-10300H
- GPU: RTX 3060 mobile
- RAM: 16 GB

# Issue: 
- Game displays a loading screen

# Screenshots:
![image](https://user-images.githubusercontent.com/106798110/172876742-f237b04c-b6d9-402c-bb9a-d17ecbb7ce5c.png)

# Log

[vita3k.log](https://github.com/Vita3K/compatibility/files/8871488/vita3k.log)

# Recommended labels
- Intro

**Comment by zyo-za** [comment]
How were you able to get past the loading screen? I also have the same problem where the game is permanently stuck at the loading screen.

**Comment by potatoteatime** [comment]
> How were you able to get past the loading screen? I also have the same problem where the game is permanently stuck at the loading screen.

Hmm, I didn't do anything special and was just able to get past the loading screen consistently from the build in the screenshot.
That said, I have since then changed PC and can't get past the loading screen as well on my new PC even on the same older build so not sure what's going on...

**Comment by zyo-za** [comment]
Yeah it used to work on my old pc too but cant get it working on my new one well that's too bad

**Comment by zyo-za** [comment]
Actually found a fix for this. I used the 3.73 version of the firmware and the vita3k with the same version in your screenshot. Loaded the game on opengl and deleted this save file ux0/user/00/savedata/PCSG01070/. Then I booted the game and now it works!

**Comment by potatoteatime** [comment]
Wow, thank you for all the research into fixing this! 
I did as you mentioned (installed 3.73 firmware, used the same version as screenshot, loaded on openGL, deleted save), but somehow it still doesn't boot for me, unfortunately. I'm happy it's working for you though! 

<img width="1088" height="601" alt="Image" src="https://github.com/user-attachments/assets/7f299604-7433-44f5-a8f1-e7eddde42fa6" />

**Comment by zyo-za** [comment]
<img width="700" height="287" alt="Image" src="https://github.com/user-attachments/assets/91964407-c417-4ddc-b974-c03c7e25e39b" />
<img width="677" height="673" alt="Image" src="https://github.com/user-attachments/assets/26d67d45-42b2-40ce-8ad7-473d7afdb9f7" />
<img width="726" height="307" alt="Image" src="https://github.com/user-attachments/assets/f5480bb8-7c75-48b8-9bb1-cb4bbd0fbff3" />
<img width="1074" height="680" alt="Image" src="https://github.com/user-attachments/assets/68bf040a-b5bd-405f-934d-c2a95d17c564" />
<img width="701" height="688" alt="Image" src="https://github.com/user-attachments/assets/a02c285f-f11e-4871-b760-a91f0ad91bd6" />
<img width="672" height="463" alt="Image" src="https://github.com/user-attachments/assets/4ba8c2bf-d427-49e1-b7f1-500a33bccd8f" />
<img width="690" height="293" alt="Image" src="https://github.com/user-attachments/assets/e2e4e307-5a89-4943-aeaf-380616155c84" />
<img width="1246" height="764" alt="Image" src="https://github.com/user-attachments/assets/2f6791ab-88b9-4efd-92e6-69be6eacd738" />
this is how my settings look

**Comment by zyo-za** [comment]
https://drive.google.com/file/d/1zxzse3Drt4kv9NbMhlMQ0goeceteNw4P/view?usp=drive_link

And try to download this firmware this is the one i used

**Comment by potatoteatime** [comment]
Thank you so much for taking the time to share your settings!
There were a few differences (e.g. use shader cache and use spir v-shader were not unchecked originally for me) so I'm not sure which were the critical settings but the game now plays for me. Much appreciated, truly! :))

<img width="931" height="549" alt="Image" src="https://github.com/user-attachments/assets/dac59324-a721-427a-ac1b-46d541aa430b" />

**Comment by zyo-za** [comment]
Yay! That's great. Have fun playing
